guides/doc-Project_API/topics/proc_working-with-hosts.adoc

```diff
@@ -0,0 +1,203 @@
+[id="working-with-hosts"]
+= Working with hosts
```

Comment:
unsure about the title; the lack of a good suggestion makes me ACK this for now. Input welcome; esp. in a follow-up PR.